samples/ResourceOwnerPasswordFlow/DeveloperNotes/ConformanceProfiles/conformance-profiles.txt

```diff
@@ -0,0 +1,4 @@
+﻿http://openid.net/wordpress-content/uploads/2015/03/OpenID-Connect-Conformance-Profiles.pdf
```

Comment:
As a related note, that would be great to test the existing Mvc sample to see how much work needs to be done to be compliant with the OIDC conformance tests :smile: